integration-tests/kyuubi-kubernetes-it/src/test/scala/org/apache/kyuubi/kubernetes/test/deployment/KyuubiOnKubernetesTestsSuite.scala

```diff
@@ -112,9 +134,14 @@ class KyuubiOnKubernetesWithClusterSparkTestsSuite
         "spark.submit.deployMode" -> "cluster",
         "spark.kubernetes.file.upload.path" -> s"hdfs://$localhostAddress:$getDFSPort/spark",
         "spark.hadoop.dfs.client.use.datanode.hostname" -> "true",
-        "spark.kubernetes.authenticate.driver.serviceAccountName" -> "spark",
-        "spark.kubernetes.driver.podTemplateFile" -> driverTemplate.getPath,
-        ZK_CLIENT_PORT_ADDRESS.key -> localhostAddress,
-        FRONTEND_THRIFT_BINARY_BIND_HOST.key -> localhostAddress)
+        "spark.kubernetes.authenticate.driver.serviceAccountName" -> "kyuubi",
+        "spark.kubernetes.driver.podTemplateFile" ->
+          s"hdfs://$localhostAddress:$getDFSPort/spark/driver.yml",
+        "spark.kyuubi.frontend.thrift.binary.bind.host" -> miniKubeIp,
```

Comment:
why not `kyuubiServerIp` ?

Reply:
Spark Driver Pod created with HostNetwork, so it's should bind on minikube ip